docs/kb/integrations/customize-semgrep-precommit.md

```diff
@@ -23,11 +23,11 @@ This example is based on the `semgrep-ci` hook, but a similar adjustment would a
 ```yaml
 repos:
 - repo: https://github.com/semgrep/pre-commit
-  rev: 'v1.101.0'
+  rev: 'v1.128.0'
   hooks:
-    - id: semgrep-verbose
+    - id: semgrep-ci
       entry: semgrep
-      args: ["ci", "--dry-run", "--baseline-commit", "HEAD" "2>/dev/null"]
+      args: ["ci", "--dry-run", "--baseline-commit", "HEAD","--quiet"]
```

Comment:
Both options are mentioned here, so what's the advantage of changing which one is provided?

Reply:
Yes good catch on the comma, but I don't think there is an issue with the redirect (see previous thread). Both are behaving correctly.